bundles/org.openhab.binding.bluetooth.radoneye/src/main/java/org/openhab/binding/bluetooth/radoneye/internal/AbstractRadoneyeHandler.java

```diff
@@ -293,7 +289,16 @@ public void onConnectionStateChange(BluetoothConnectionStatusNotification connec
     private boolean isTimeToRead() {
         int sinceLastRead = sinceLastReadSec.get();
         logger.debug("Time since last update: {} sec", sinceLastRead);
-        return sinceLastRead >= refreshInterval;
+        return sinceLastRead >= configuration.refreshInterval;
+    }
+
+    /**
+     * Provides the configuration of this instance
+     *
+     * @return the configuration of this instance
+     */
+    protected @Nullable RadoneyeConfiguration getConfiguration() {
```

Comment:
With the change I proposed below, you can then remove `@Nullable` here.

Considering how this method is used, I would suggest to replace it by a method getFwVersion.